Dockerfile

```diff
@@ -1,33 +1,35 @@
-FROM golang:1.17-alpine
+FROM golang:1.17-alpine as base
 MAINTAINER team@pganalyze.com
 
-RUN adduser -D pganalyze pganalyze
 ENV GOPATH /go
 ENV HOME_DIR /home/pganalyze
 ENV CODE_DIR $GOPATH/src/github.com/pganalyze/collector
```

Comment:
Not related to this change, but any thoughts whether we should use `ARG` here instead of `ENV`? (I guess it doesn't matter since the multi-stage build now hides the env settings here anyway)

Reply:
ARGs are only really useful if you plan to pass them as parameters to the `docker build` step, otherwise ENV var is the route to go imo.